apps/events/migrations/0011_no_activity_pings_to_events_20240619_1436.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 4.2.11 on 2024-06-19 14:36
+
+from django.db import migrations
+
+from apps.events.models import EventActionType
+
+
+def convert_no_activity_pings_to_event(apps, schema_editor):
+    NoActivityMessageConfig = apps.get_model("experiments", "NoActivityMessageConfig")
+    TimeoutTrigger = apps.get_model("events", "TimeoutTrigger")
+    EventAction = apps.get_model("events", "EventAction")
+
+    for no_activity_ping in NoActivityMessageConfig.objects.all():
```

Comment:
Nit: use `prefetch_related`. I see there are only 18 records in prod, so not really necessary :man_shrugging: . Up to you